### ISSUE TITLE
Regarding about the epoch and batch size for image size 320

### ISSUE THREAD

**Issue by greattester640** [surendramaran/YOLOv8-TfLite-Object-Detector#27]
Hello, I saw the previous comment about the recommended Yolov8 models. You did mention about the image size 320. When it comes to image size 320, how many epochs and batch size do you recommended? Or test until it is good?

**Comment by surendramaran** [comment]
Yeah the image size will depend, lesser the resolution lesser the accuracy more speed up.

I never considered batch size for this repo, so can't say anything about it now. In general, there's a trade-off between batch size, inference speed, memory usage, and application requirements (like real-time performance)

**Comment by greattester640** [comment]
Ok, noted.
